packages/components/buttons/secondary-button/src/secondary-button.tsx

```diff
@@ -65,8 +66,13 @@ export type TSecondaryButtonProps<
   size?: 'medium' | 'big';
   /**
    * Indicates the color scheme of the button.
+   * @deprecated Use `tone` instead.
```

Comment:
Please, remember we also want to remove the knob from the Storybook story for this property.

Reply:
Thanks for the reminder [c40fe94](https://github.com/commercetools/ui-kit/pull/2576/commits/c40fe941eeaf41d2d1deeb83da22946372329b88)